hack/lib/util.sh

```diff
@@ -322,7 +335,7 @@ kube::util::git_upstream_remote_name() {
 kube::util::create-fake-git-tree() {
   local -r target_dir=${1:-$(pwd)}
 
-  pushd "${target_dir}" >/dev/null
+  pushd "${target_dir}" >/dev/null || return
```

Comment:
Why would we want || return here? If we can't cd into target dir, I would expect this to error out

Reply:
We can error out too. I'll change this. Probably the implicit assumption has been that `util.sh` is anyway sourced from a file which has set `errexit`.

Reply:
Fixed.